### PR TITLE
Add date to card

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: src/main/java/core/selenium/util/DateMethod.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import java.util.Map;
+
+/**
+ * DateMethod class.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public final class DateMethod {
+    private static Map<String, Integer> hours = new HashMap<>();
```
Comment (raulchoque-eng) [inline src/main/java/core/selenium/util/DateMethod.java:30]: Please, add a white space above this line.

File: src/main/java/trello/ui/pages/BoardPage.java [hunk 2]
```diff
@@ -100,7 +101,7 @@
      * Constructor method for init parameter of this class.
      */
     public BoardPage() {
-        listForm = new ListForm();
+//        listForm = new ListForm();
     }
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/BoardPage.java]: Please, delete the method constructor .

File: src/main/java/trello/ui/pages/modal/CardModal.java [hunk 3]
```diff
@@ -42,18 +44,88 @@
...
+    private WebElement dateButton;
+
+    @FindBy(className = "js-change-card-members")
+    private WebElement memberButton;
+
+    @FindBy(className = "js-card-detail-members-list")
+    private WebElement cardMembersList;
+
+//    @FindBy(className = "js-member-on-card-menu")
+//    private List<WebElement> cardMemberItem;
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/modal/CardModal.java]: Please, clear or decomment these lines.

File: src/test/resources/features/card.feature [hunk 4]
```diff
@@ -13,3 +13,24 @@ Feature: Card actions
...
+  Scenario: Assign card to a member
+    Given I log in as admin user
+     When I select TestBoard board
+      And I select TestCard card
+      And I add a member
+        | George Smith  |
+        | James Cooper  |
+        | Joseph Taylor |
+     Then the member's initials are shown in the card
+#      And the member is checked on the board members list
```
Comment (raulchoque-eng) [inline src/test/resources/features/card.feature]: Please, cleat or decomment this line.
Add a white line bellow this line, please.

File: src/main/java/core/selenium/util/DateMethod.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/util/DateMethod.java:30]: Solved

File: src/main/java/trello/ui/pages/BoardPage.java (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/trello/ui/pages/BoardPage.java]: Solved

File: src/main/java/trello/ui/pages/modal/CardModal.java (hunk 3 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/trello/ui/pages/modal/CardModal.java]: Solved

File: src/test/resources/features/card.feature (hunk 4 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/features/card.feature]: Solved

**Pull request closed without merging**